src/stories/containers/Finances/SeccionPages/BreadcrumbYearNavigation.stories.tsx

```diff
@@ -0,0 +1,57 @@
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import BreadcrumbYearNavigation from './BreadcrumbYearNavigation';
+import type { NavigationBreadCrumb } from '@ses/components/Breadcrumbs/Breadcrumbs';
+import type { ComponentMeta } from '@storybook/react';
+import type { FigmaParams } from 'storybook-addon-figma-comparator/dist/ts/types';
+
+export default {
+  title: 'Pages/NewFinances/BreadcrumbYearNavigation',
+  component: BreadcrumbYearNavigation,
+
+  parameters: {
+    chromatic: {
+      viewports: [1440],
+      pauseAnimationAtEnd: true,
+    },
+  },
+} as ComponentMeta<typeof BreadcrumbYearNavigation>;
+
+const args = [
+  {
+    years: ['2022', '2023'],
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    handleChange: () => {},
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    onOpen: () => {},
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    onClose: () => {},
```

Comment:
use `() => null` instead, this way you don't need to disable eslint checks

Reply:
done